packages/nimble-components/src/table/styles.ts

```diff
@@ -89,16 +83,77 @@ export const styles = css`
         visibility: visible;
     }
 
+    .header-container {
+        display: flex;
+        align-items: center;
+        position: relative;
+    }
+
     .header-scrollbar-spacer {
         width: var(--ni-private-table-header-scrollbar-spacer-width);
     }
 
     .header {
         flex: 1;
+        overflow: hidden;
     }
 
-    .checkbox-container {
-        display: flex;
+    .column-divider {
+        border-left: 2px solid ${popupBorderColor};
+        display: none;
+        height: ${controlSlimHeight};
+        cursor: col-resize;
+        position: absolute;
+    }
+
+    .column-divider::before {
+        content: '';
+        position: absolute;
+        width: 8px;
+        height: 100%;
+        left: -5px;
```

Comment:
Where does `-5px` come from? Is it possible to express it as a calculation of other named values?